CHANGES_NEXT_RELEASE

```diff
@@ -3,3 +3,5 @@ Fix: Add multiservice flag for the default configuration in the RPM. (Issue #964
 Add: New operation "GET /v2/entities" (Issue #947)
 Add: New operation "GET /v2" (Issue #956)
 Fix: ONCHANGE subscription sends notification when value under condition it is not updated with updateContext-APPEND in some cases (Issue #943)
+Add: Reuse curl context for outgoing notifications
```

Comment:
I would suggest to be more "functionality oriented", i.e.

"Add: Reuse curl context for outgoing notifications, so now connections are persistent if the HTTP server doesn't close them"

Or something like that...